### PR TITLE
Roll qpid-proton within the router image build eliminating the need for the dependency on the gordons/qpid-proton:0.23 image.

### PR DESCRIPTION
**Pull request by k-wall**

### CODE REVIEW

**Comment by grs** [comment]
The two patches in there are not currently used and can be removed. The VERSION tag in .travis.yml should probably be altered to produce a new label (e.g. 1.2.0-PROTON-1886).

**Comment by grs** [comment]
Otherwise looks good to me. Nice work!

File: build_tarballs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/bash
+
+
+do_patch () {
+    PATCH_DIR=$1
+    PATCH_SRC=$2
+    if [ -d "${PATCH_DIR}" ]
+    then
+        for patch in $(find ${PATCH_DIR} -type f -name "*.patch"); do
```
Comment (grs) [inline build_tarballs]: find not found in travis build output

**Comment by k-wall** [comment]
@grs's comments addressed.

**Comment by lulf** [comment]
@k-wall @grs Should we consider building this image off CentOS as the other images? Basing on fedora I think has a disadvantage that security fixes are not supported for long. Fedora 25 is a while ago, so consider at least updating it to 28.

I'm fine leaving this as is and doing that work as another task.

**Comment by grs** [comment]
@lulf, I agree with switching to centos (or failing that upgrading to later fedora). Also agree it can be done as a separate PR.

**Comment by k-wall** [comment]
@lulf yes, consistency is good, so I'd prefer CentOS, but let me raise a separate PR.

**Comment by k-wall** [comment]
On the switching to CentOS, if websocket support is required, Dispatch Router requires libwebsockets 2.1.0 or higher (see [DISPATCH-893](https://issues.apache.org/jira/browse/DISPATCH-893)).  Unfortunately, the CentOS 7 makes only libwebsockets-devel-1.7.5-1.el7.x86_64 available (from the optional EPEL repo).  I think we will need to go with a later Fedora instead.

**Pull request merged**